src/tests/fuzztests/xi_fuzztest_mqtt_layer.cpp

```diff
@@ -0,0 +1,10 @@
+#include <memory.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <string.h>
+
+extern "C" int LLVMFuzzerTestOneInput( const uint8_t* data, size_t size )
```

Comment:
ah is this a function which has to be implemented and executed by the fuzzer right? so you cannot determine the naming, then it is ok, sorry.